runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -115,3 +110,175 @@ task_t<int64_t> f$openssl_verify(const string &data, const string &signature, co
 
   co_return 1;
 }
+
+Optional<tl::CipherAlgorithm> parse_algorithm(const string &method) {
```

Comment:
Please, add `noexcept` to all the functions

Reply:
fux